src/scss/components/_list.scss

```diff
@@ -55,29 +52,37 @@
     display: block;
 
     .list__item {
-      display: block;
       width: auto;
       float: none;
+    }
 
-      &::before {
-        margin-right: 2px;
-      }
+    &[class*='col-'] {
+      display: block;
     }
   }
 
   &__item {
-    @include flex-shrink(0);
-    @include flexbox;
-
-    padding: 8px 0;
```

Comment:
I know that, if we decided to use padding, text will break the line and subsequent 😭